service/http.redirect/src/main/java/org/kaazing/gateway/service/http/redirect/HttpRedirectService.java

```diff
@@ -47,8 +48,21 @@ public void setConfiguration(Properties configuration) {
 
     @Override
     public void init(ServiceContext serviceContext) throws Exception {
+        logger.info("Entering http.redirect init");
         EarlyAccessFeatures.HTTP_REDIRECT.assertEnabled(configuration, serviceContext.getLogger());
 
+        Collection<String> acceptURIs = serviceContext.getAccepts();
+        for (String acceptURI : acceptURIs) {
+            logger.info("acceptURI=" + acceptURI);
```

Comment:
Please do not log any of this, especially at info level: https://docs.google.com/document/d/10IYGNlvqRIfqfseQ36M4uIA8P-dgbZkQ__wbHaGiIZw/edit#heading=h.ate0rhjaq47f.  

Trace is okay if you really feel like it is needed, but 

1) The exception will already be thrown and logged, so need to put "Throwing Exception"
2) The accept URIs for all the services are already logged at startup, so no need for that.  
3) Entering http.redirect init will be in the stack trace of any exception thrown here...